plugin/src/main/java/com/epam/grandhackathon/deployment/sphere/plugin/DeployVersionMetaDataCollectorDescriptor.java

```diff
@@ -45,4 +54,13 @@ public FormValidation doCheckDeployedAppName(@QueryParameter String deployedAppN
         }
         return FormValidation.ok();
     }
+    
+    public ListBoxModel doFillDeployedAppNameItems() {
```

Comment:
Whe have choiser in 2 place Build job and Deploy job. And these methods should have name doFillxxxItems  and be in Descriptor.